### PR TITLE
Add cluster-kube-apiserver-operator plugins

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sttts** [comment]
@smarterclayton @stevekuznetsov can you approve?

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -261,6 +261,11 @@ plugins:
   openshift/cluster-capacity:
   - trigger
 
+  openshift/cluster-kube-apiserver-operator:
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/plugins.yaml]: After #1229  this PR is not needed anymore, will happen by default

**Comment by stevekuznetsov** [comment]
Oops, didn't see you also added to `tide` config
/reopen

**Comment by sttts** [comment]
> Oops, didn't see you also added to tide config
> /reopen

So who can and will merge this?

**Comment by sttts** [comment]
/assign @smarterclayton 

for approval.

**Comment by sttts** [comment]
Fixed the plugin conflict.

**Comment by petr-muller** [comment]
Yeah it's benign here, and it does not provide any useful information right now. I have submitted a PR to not run it on `\test all` [here](https://github.com/openshift/release/pull/1287).

**Pull request merged**